README.md

```diff
@@ -260,8 +260,8 @@ and disables the auto-detection feature:
 
 Parameter | Type | User Property | Required | Description
 --- | --- | --- | --- | ---
-`<chartDirectory>` | string | helm.chartDirectory | true | root directory of your charts
-`<chartVersion>` | string | helm.chartVersion | true | Version of the charts. The version have to be in the [SEMVER-Format](https://semver.org/), required by helm.
+`<chartDirectory>` | string | helm.chartDirectory | false | root directory of your charts
+`<chartVersion>` | string | helm.chartVersion | false | Version of the charts. The version have to be in the [SEMVER-Format](https://semver.org/), required by helm.
```

Comment:
Q: Why are these being changed from required to optional?  (From `AbstractChartDirectoryMojo`, it actually looks like chartDirectory _is_ required.)

Reply:
Does it hurt anything to leave it though? Were you running into an issue or was it just conceptual disagreement with the structure of the code?

Reply:
Was about to follow up, but I think this is a moot point since we want to use the chart directory parameter for the planned extension now, right?

Reply:
No we dont. We want to be able to call this plugin on say the root aiSSEMBLE dir. The plugin is called for each maven project under that directory. It determines if the project contains a Helm chart and pushes its chart dependencies. We dont want to send in the chartDirectory path since it changes between each call on the plugin.